src/topics/group-generator/group-generator.ts

```diff
@@ -251,7 +251,10 @@ export class GroupGeneratorService {
         groupId: newId,
         timestamp: group.timestamp,
         name: group.name,
-        properties: this.computeProperties(group.data) as Properties,
+        properties:
+          group.name === "sismo-contributors"
+            ? (this.computeProperties(group.data) as Properties)
```

Comment:
You need to keep the computeProperties but just not do the `valueDistribution` 
The computeProperties also compute the accountNumber propertie